nemo/collections/asr/parts/utils/rnnt_utils.py

```diff
@@ -74,6 +76,7 @@ class Hypothesis:
     score: float
     y_sequence: Union[List[int], torch.Tensor]
     text: Optional[str] = None
+    text_all: Optional[str] = None
```

Comment:
Can you reuse y_sequence rather than add another argument to this?

Reply:
dropped it.